### PR TITLE
Add NVIDIA integration tile

### PR DESCRIPTION
**Pull request by annthurium**

### CODE REVIEW

File: integrations/nvidia.md
Comment (bilgeyucel) [inline integrations/nvidia.md:12]: We need to provide links for the integration. Also below in `report_issue` field

File: integrations/nvidia.md
Comment (bilgeyucel) [inline integrations/nvidia.md:29]:
```suggestion
This integration allows you to use NVIDIA Foundation Models in your Haystack pipelines.
```

File: integrations/nvidia.md
Comment (bilgeyucel) [inline integrations/nvidia.md:122]:
```suggestion
from haystack_integrations.components.embedders.nvidia import NvidiaEmbeddingModel, NvidiaDocumentEmbedder
from haystack import Pipeline
```

File: integrations/nvidia.md
Comment (bilgeyucel) [inline integrations/nvidia.md:44]: We dont have any docs page, right?

File: integrations/nvidia.md
Comment (bilgeyucel) [inline integrations/nvidia.md:85]: Do users usually know which model this `NVOLVE_40K` refers to?

File: integrations/nvidia.md
Comment (silvanocerza) [inline integrations/nvidia.md:14]:
```suggestion
pypi: https://pypi.org/project/nvidia-haystack
repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/nvidia
type: Model Provider
report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
```

File: integrations/nvidia.md
Comment (silvanocerza) [inline integrations/nvidia.md:29]:
```suggestion
This integration allows you to use NVIDIA Foundation Models in your Haystack pipelines.
```

File: integrations/nvidia.md
Comment (annthurium) [reply integrations/nvidia.md:44]: correct - we don't have a docs page yet.

**Comment by shadeMe** [comment]
FYI: Let's not merge this PR until later this week - We'll be updating the integration to support a new Nvidia service, which will change the API a bit. Once those changes have been merged into core integrations, we'll update the code snippets above and sign off this PR's merging.

File: integrations/nvidia.md
Comment (annthurium) [reply integrations/nvidia.md:85]: it's this one - https://catalog.ngc.nvidia.com/orgs/nvidia/teams/ai-foundation/models/nvolve-40k

Adding a link is probably a good idea just to make that clearer.

**Comment by annthurium** [comment]
@shadeMe works for me as long as we have some sort of page up by next Monday when I head to GTC.

File: integrations/nvidia.md
Comment (shadeMe) [inline integrations/nvidia.md:165]:
```suggestion
text_embedder = NvidiaTextEmbedder(model="nvolveqa_40k")
```

**Comment by TuanaCelik** [comment]
@annthurium @shadeMe Should we be merging this now? What's the situation?

**Review by TuanaCelik — APPROVED** [review]

**Pull request merged**
